src/Compiler/Service/IncrementalBuild.fs

```diff
@@ -127,7 +127,7 @@ module IncrementalBuildSyntaxTree =
                 use act =
                     Activity.start "IncrementalBuildSyntaxTree.parse"
                         [|
-                            "fileName", source.FilePath
+                            Activity.Tags.fileName, source.FilePath
                             "buildPhase", BuildPhase.Parse.ToString()
                             "canSkip", canSkip.ToString()
```

Comment:
Should these also go to `Activity.Tags`?

Reply:
They were only used at one place, they do not meet the "common enough" bar to make it.